src/main/java/com/gigacal/specifications/CalendarSpecification.java

```diff
@@ -0,0 +1,11 @@
+package com.gigacal.specifications;
+
+import com.gigacal.entity.CalendarEntity;
+import org.springframework.data.jpa.domain.Specification;
+
+public class CalendarSpecification {
+
+    public static Specification<CalendarEntity> hasName(String name){
```

Comment:
If you would simply add the method findByName() into the CalendarRepository you would avoid adding additional class just to get the calendar with a specific name